config/emendatusenigmatica/strata/stone.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This should be expanded to account for any other stone types we have underground. Create, vanilla stone variants like granite and andesite, etc. 

Keep Blue Skies and Twilight Forest in mind here too, as they should also be unified under EE.